views/mixins/overviewFields.jade

```diff
@@ -0,0 +1,10 @@
+mixin overviewDetails(fields)
+	each fieldset, key in fields
+		section.group
+			h1=key
+			div.body
+				each field in fieldset
+					div.field
+						p
+							span.col-sm-2.label=field.label
```

Comment:
add row class to div that wraps fields. container class is already added to the outer most div.